backend/users/models.py

```diff
@@ -1,11 +1,13 @@
 from django.db import models
 
 # Create your models here.
-from django.contrib.auth.models import AbstractUser
 
-class CustomUser(AbstractUser):
-    EMAIL_FIELD = 'email'
-    USERNAME_FIELD = 'email' 
-    REQUIRED_FIELDS = ['username']
-    username = models.CharField(blank=True, max_length=254,)
-    email = models.EmailField(blank=False, max_length=254, unique=True, verbose_name='email address')
\ No newline at end of file
+class Event(models.Model):
+    name = models.CharField(max_length=100)
+    date = models.DateTimeField()
+    tag = models.CharField(max_length=100)
+    organizer = models.CharField(max_length=100)
+    location = models.CharField(max_length=100)
```

Comment:
Would having a  `description` as well make sense? You can think and decide about it.

Reply:
Added description data type in commit 5c07a74f41277987ca66d029f8b1b310e674a92a